### ISSUE TITLE
handle slackmoji

### ISSUE THREAD

**Issue by zetlen** [HurrSlack/pinwheel#10]
the slack API sends messages with raw formatting, like *bold* and _italic_. this is probably fine, but emoji come through as `:100:` instead of 💯 , so we should process them only if they're part of the emoji set that twitter supports

**Comment by jakestenger** [comment]
How would we handle this? A manual mapping of the slack shortcut to the twitter shortcut in a JSON? Just grep for :*: in the string and if an instance is found reference the mapping file and make the replacement?

**Comment by zetlen** [comment]
yeah i think that's basically it, though the regex would probably be more like `\b:\w\-:\b`.

**Comment by charlesvestal** [comment]
Another option is to use the screens hotting whenever any emoji are detected

**Comment by zetlen** [comment]
Yeah, but only when emoji that twitter doesn't already support are detected. There's an endpoint where you can get a list of the installed slackmoji, and we could make a big regex out of it and test...

**Comment by charlesvestal** [comment]
i wonder if it's worth it for consistency tho? (but if so, we'll also we'll need to adjust the trimming, as the trimmed text includes :emojinames:)

**Comment by zetlen** [comment]
i like the idea of text posts where text posts are possible, so going straight from slackmoji to twitmoji feels right when it's possible. it's not like it would be common--most emoji we use are our own
